dolly-backend-app/src/main/java/no/nav/dolly/bestilling/organisasjonforvalter/OrganisasjonClient.java

```diff
@@ -116,12 +116,17 @@ private void saveOrgnumreToDbAndDeploy(Set<String> orgnumre, Long bestillingId,
         ResponseEntity<DeployResponse> deployResponse = organisasjonConsumer.deployOrganisasjon(new DeployRequest(orgnumre, environments));
 
         if (deployResponse.hasBody()) {
-            requireNonNull(deployResponse.getBody()).getOrgStatus().entrySet().forEach(orgStatus -> organisasjonProgressService.save(OrganisasjonBestillingProgress.builder()
-                    .bestillingId(bestillingId)
-                    .organisasjonsnummer(orgStatus.getKey())
-                    .organisasjonsforvalterStatus(mapStatusFraDeploy(orgStatus))
-                    .uuid(mapUuidFraDeploy(orgStatus))
-                    .build()));
+            requireNonNull(deployResponse.getBody()).getOrgStatus().entrySet().forEach(orgStatus -> {
```

Comment:
Ser ikke hva requireNonNull gjør xtra her. 
.hasBody() gjør null-sjekken for deg.

Reply:
![image](https://user-images.githubusercontent.com/66469591/105141205-17590a80-5af9-11eb-8971-fdcce185052c.png)
Ja helt enig, men hvis jeg fjerner den så klager sonar 😅